scripts/test/educator.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Runs educator tests.
+# All passed parameters go to --test-arguments parameter of the test executable.
+
+# Dependencies:
+# * Postgres server
+# * pg_tmp (http://eradman.com/ephemeralpg/)
```

Comment:
Can we add those to `shell-native.nix`?

Reply:
@yorickvP do I understand correctly that you will need to do this for CI's sake anyway? :roll_eyes: